esm/interface/element.js

```diff
@@ -75,11 +75,19 @@ const isVoid = ({localName, ownerDocument}) => {
  * @implements globalThis.Element
  */
 export class Element extends ParentNode {
-  constructor(ownerDocument, localName) {
+  constructor(ownerDocument, localName, nsp) {
     super(ownerDocument, localName, ELEMENT_NODE);
     this[CLASS_LIST] = null;
     this[DATASET] = null;
     this[STYLE] = null;
+    this[NS] = nsp
+  }
+
+  /**
+   * @returns {String}
+   */
+  get namespaceURI() {
+    return this[NS]
```

Comment:
as written before, this is where `instanceof` check seems ot be more appropriate for *SVG* elements ... I myself never needed any of this in the wild, so let's make it lazy, please?